recipes/feather-format/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: feather-format
+  version: "0.2.0"
+
+source:
+  fn: feather-format-0.2.0.tar.gz
+  url: https://pypi.python.org/packages/06/bb/072a00246f235df6a902dbaf4cb10d687ea5b3a199f786dd99c23cd08cf8/feather-format-0.2.0.tar.gz
```

Comment:
Can be changed to this [https://pypi.io/packages/source/f/feather-format/feather-format-0.2.0.tar.gz]( https://pypi.io/packages/source/f/feather-format/feather-format-0.2.0.tar.gz ). Sorry it is a pane. Blame PyPI for the mess.

With that done, could use jinja for the versions if you like. Might make things easier down the road.